techniques/general/G4.html

```diff
@@ -51,86 +49,85 @@
-   </section><section id="examples"><h2>Examples</h2>
+<section id="description"><h2>解説</h2>
+      <p>この達成方法の目的は、コンテンツの動きやスクロールの一時停止方法を提供することである。利用者が、動きを一時停止させる、注意をそらすものを減らす、又は読む時間を作る必要があるのであれば、そうすることができ、さらに必要に応じて動きやスクロールを再開できるようにする。このメカニズムは、WCAG に準拠したインタラクティブなコントロール又はキーボードショートカットを介して提供できる。もしキーボードショートカットを用いるのであれば、ウェブページで利用者に説明する。</p>
+   </section><section id="examples"><h2>事例</h2>
       <ul>
-         <li>A site contains a scrolling news banner at the top of the page. Users who need more time to read it can press the Escape key to pause the scrolling. Pressing Escape again restarts it.</li>
-         <li>A web page contains a link labeled "How to tie a shoe" which links to an animation. Text  immediately preceding the link informs the user that pressing the spacebar will pause the animation and restart it again.</li>
+         <li>ウェブページの上部に、スクロールするニュースのバナーが表示されている。これを読むのにより多くの時間が必要な利用者は、Esc キーを押すと、スクロールを一時停止できる。そして、Esc キーをもう一度押すと、スクロールが再開する。</li>
+         <li>ウェブページには、Flash アニメーションにリンクする「靴紐の結び方」というリンクがある。リンクの直前のテキストは、スペースキーを押すとアニメーションを一時停止し、もう一度押すと再開することを利用者に知らせる。</li>
       </ul>
    </section>
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../general/G75">G75: Providing a mechanism to postpone any updating of content</a></li>
       <li><a href="../general/G76">G76: Providing a mechanism to request an update of the content instead of updating automatically</a></li>
       <li><a href="../general/G186">G186: Using a control in the web page that stops moving, blinking, or auto-updating content</a></li>
       <li><a href="../client-side-script/SCR33">SCR33: Using script to scroll content, and providing a mechanism to pause it</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
-         <p>On a page with moving or scrolling content,</p>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
+         <p>動き又はスクロールのあるコンテンツがあるウェブページ上で:</p>
          <ol>
-            <li>Use the mechanism provided in the web page or by the user agent to pause the moving or scrolling content.</li>
-            <li>Check that the moving or scrolling has stopped and does not restart by itself.</li>
-            <li>Use the mechanism provided to restart the moving content.</li>
-            <li>Check that the movement or scrolling has resumed from the point where it was stopped.</li>
+            <li>ウェブページで、又はユーザエージェントによって提供されているメカニズムを用いて、コンテンツの動き又はスクロールを一時停止させる。</li>
+            <li>その動き又はスクロールが停止し、勝手に再開しないことを確認する。</li>
+            <li>提供されているメカニズムを用いて、コンテンツの動きを再開する。</li>
+            <li>動き又はスクロールが、停止したところから再開することを確認する。</li>
```

Comment:
Check that ... の訳を、他のテクニックと合わせました。
```suggestion
            <li>動き又はスクロールが、停止したところから再開することをチェックする。</li>
```